IEEEKeralaSection/rescuekerala
# API for Relief Camp Coordinators App

### Summarize
Provides an API for the Relief Camp Coordinators android application.

APIs for:
- Fetch list of relief camps by district
- Person registration to a camp
- Update relief camp supply requirements

Technical Changes:
- Adds django-rest-framework and jwt-auth modules
- api_urls.py and api_views.py for corresponding routes and controllers

Application:
The application is developed in a different repo; testing instructions available there
[Repo link](https://github.com/Appmaker-xyz/KeralaRescueRefugeeDB)

Testing instructions:
- Test existing features (mainly those involving login) to make sure nothing else is affected (very low chance)
- Testing the API endpoints via the application. See [app's repo](https://github.com/Appmaker-xyz/KeralaRescueRefugeeDB) for instructions
- Making sure data entered through the app appears correctly on website/DB

The app is planned to go live before noon on 19-Aug-2018.
